node/pkg/admin/tests/main_test.go

```diff
@@ -71,6 +74,12 @@ func insertSampleData(ctx context.Context) (*TempData, error) {
 	}
 	tempData.feed = tmpFeed
 
+	tmpProxy, err := db.QueryRow[proxy.ProxyModel](ctx, proxy.InsertProxy, map[string]any{"protocol": "http", "host": "localhost", "port": 80, "location": "test"})
+	if err != nil {
+		return nil, err
+	}
+	tempData.proxy = tmpProxy
```

Comment:
@nick-bisonai It is always good to unify `temp` vs `tmp`.

Reply:
I'll make an update